packages/puppeteer-core/src/api/Page.ts

```diff
@@ -2482,12 +2236,10 @@ export abstract class Page extends EventEmitter<PageEvents> {
    *
    * @experimental
    *
-   * @privateRemarks BiDi
-   * {@link PROTOCOL_GET_STARTED_LINK_TEMPLATE | CDP}: Supported
-   *
-   * {@link PROTOCOL_GET_STARTED_LINK_TEMPLATE | BiDi}: Unsupported
+   * @remarks
    *
-   * {@link PROTOCOL_GET_STARTED_LINK_TEMPLATE | BiDi+}: Supported
+   * All recordings will be {@link https://www.webmproject.org/ | WebM} format using
+   * the {@link https://www.webmproject.org/vp9/ | VP9} video codec. The FPS is 30.
```

Comment:
This line `You must have {@link https://ffmpeg.org/ | ffmpeg} installed on your system.` is missing after this change. Shall we add it back?

Reply:
Yeah, it got lost by accident